10up/ElasticPress
# Update index on Attach/detach media

### Description of the Change
This PR fill fix the issue where attaching or detaching the media is not updating the index. 



Closes #3542 

### How to test the Change

- Enable Protected content from ElasticPress settings
- Go to the media library and Select the view mode as a list.
- Attach/Detach the media
- Check that the attachments are coming from ES with updated parent. 


### Changelog Entry

> Fixed - Update index on attach/detach media


### Credits

Props @MARQAS 

### Checklist:
